source/views/contacts/list.tsx

```diff
@@ -12,7 +12,7 @@ import type {ContactType} from './types'
 
 const styles = StyleSheet.create({
 	listContainer: {
-		backgroundColor: c.white,
+		backgroundColor: c.systemBackground,
```

Comment:
I'm still not sure when to use these semantic colors - is this systemBackground, or secondarySystemGroupedBackground?